app/scripts/lib/channels/receivers/postmessage.js

```diff
@@ -70,14 +70,11 @@ define(function (require, exports, module) {
         err.context = origin;
 
         this.trigger('error', err);
-        return;
```

Comment:
Part of the problem is we (we meaning __I__) failed to add a return after line 62. Instead of returning all over the place, have the happy path contained in the final `else` and eliminate the other `return` statements.